### PR TITLE
Update sf pkg

### PR DESCRIPTION
**Pull request by vivigirardin**
The app is crashing when the user clicks on `show <waterbody>`. When I started to dig into the issue, it seemed like the problem was with the Shiny version. Once I updated it, then I realized that the package `rdgal` is [no longer in cran](https://cran.r-project.org/web/packages/rgdal/index.html). 

This PR updates Shiny to its latest version and replaces `rdgal` with [sf](https://r-spatial.github.io/sf/)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM rocker/shiny:4.0.0
+FROM rocker/shiny:latest
```
Comment (knl88) [inline Dockerfile:1]: perhaps you can specify the shiny version you have used, using `latest` could break in a couple of years:)

File: Dockerfile [hunk 2]
```diff
@@ -8,13 +8,14 @@ RUN apt-get update -y && \
...
     libspatialindex-dev \
+    libudunits2-dev \
     proj-bin \
     proj-data
 
 ENV CPLUS_INCLUDE_PATH /usr/include/gdal
 
 # Install R packages
-RUN install2.r --error --ncpus 4 --deps TRUE tidyverse shiny leaflet rgdal raster shinydashboard DT shinyjs
+RUN install2.r --error --ncpus 4 --deps TRUE dplyr shiny leaflet raster shinydashboard DT shinyjs sf
```
Comment (knl88) [inline Dockerfile:18]: you can use the `dependecies.R` file instead here

Comment (vivigirardin) [reply Dockerfile]: 1e953783542b6a52710f05bbc0cb9dc4dca6ab7b

**Review by knl88 — APPROVED** [review]
lgtm:)

**Pull request merged**